### PR TITLE
Added received messages metric to listener

### PR DESCRIPTION
**Pull request by Josca**
- received messages metric
- evaluation duration metric
- updates count metric, etc

### CODE REVIEW

File: database/schema/create_schema.sql [hunk 1]
```diff
@@ -587,6 +588,7 @@ CREATE TRIGGER system_advisories_set_first_reported BEFORE INSERT ON system_advi
   FOR EACH ROW EXECUTE PROCEDURE set_first_reported();
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON system_advisories TO evaluator;
+GRANT SELECT, INSERT, UPDATE, DELETE ON system_advisories TO listener;
```
Comment (semtexzv) [inline database/schema/create_schema.sql]: We'll have to remove this permissions once we separate evaluator from listener.

File: listener/upload.go [hunk 2]
```diff
@@ -18,16 +18,25 @@ import (
 func uploadHandler(event PlatformEvent) {
 	if event.B64Identity == nil {
 		utils.Log().Error("Identity not provided")
+		messagesReceivedCnt.WithLabelValues("error-identity").Inc()
```
Comment (semtexzv) [inline listener/upload.go]: Please add prefix or separate metrics to differentiate between upload & delete messages.

File: database/schema/create_schema.sql (hunk 1 above)
Comment (Josca) [reply database/schema/create_schema.sql]: Yes, but it wouldn't be too difficult :smile:

File: listener/upload.go (hunk 2 above)
Comment (Josca) [reply listener/upload.go]: ok, done

**Pull request merged**